src/utils/formatAuthorName.ts

```diff
@@ -0,0 +1,14 @@
+const formatAuthorName = (authorName: string | string[]) =>
+  Array.isArray(authorName)
+    ? `[${authorName.map(
+        name => `{
+      "@type": "Person",
+      "name": "${name}"
+    }`,
```

Comment:
I think we need to remove the trailing `,` or else it will fail validation on [google structured testing tool](https://search.google.com/structured-data/testing-tool/u/0/). 🤔

Reply:
I think this trailing comma is not problem because it is not part of the returned string, but I have updated the format of the code for easier reading.